packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/Confirm/template.success.tsx

```diff
@@ -47,6 +47,18 @@ const Success = (props: Props) => {
     ? (intervalToDuration({ end: props.withdrawLockCheck?.lockTime || 0, start: 0 }).days as number)
     : 0
 
+  const targetCoinName = useMemo(() => {
+    const currency = props.defaultMethod?.currency
```

Comment:
`const { currency } = props.defaultMethod` serves the same purpose. If not present, will be undefined. (But `defaultMethod` should exist)

Reply:
typescript dont let me deconstruct the defaultMethod here because the defaultMethod can be undefined and that could throw an error when trying to deconstruct the undefined value